[2025-04-28T06:45:24.466+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Airflow_ETL_DAG.insert_locations scheduled__2025-03-10T00:00:00+00:00 [queued]>
[2025-04-28T06:45:24.476+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Airflow_ETL_DAG.insert_locations scheduled__2025-03-10T00:00:00+00:00 [queued]>
[2025-04-28T06:45:24.476+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-04-28T06:45:24.494+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): insert_locations> on 2025-03-10 00:00:00+00:00
[2025-04-28T06:45:24.503+0000] {standard_task_runner.py:57} INFO - Started process 363 to run task
[2025-04-28T06:45:24.509+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'Airflow_ETL_DAG', 'insert_locations', 'scheduled__2025-03-10T00:00:00+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/airflow_etl.py', '--cfg-path', '/tmp/tmpo5jyuk7d']
[2025-04-28T06:45:24.512+0000] {standard_task_runner.py:85} INFO - Job 3: Subtask insert_locations
[2025-04-28T06:45:24.613+0000] {task_command.py:416} INFO - Running <TaskInstance: Airflow_ETL_DAG.insert_locations scheduled__2025-03-10T00:00:00+00:00 [running]> on host 77b690a9fde8
[2025-04-28T06:45:24.767+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='Airflow_ETL_DAG' AIRFLOW_CTX_TASK_ID='insert_locations' AIRFLOW_CTX_EXECUTION_DATE='2025-03-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-10T00:00:00+00:00'
[2025-04-28T06:45:25.810+0000] {logging_mixin.py:151} INFO - Insertion des locations terminée.
[2025-04-28T06:45:25.814+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-04-28T06:45:25.835+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=Airflow_ETL_DAG, task_id=insert_locations, execution_date=20250310T000000, start_date=20250428T064524, end_date=20250428T064525
[2025-04-28T06:45:25.933+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-04-28T06:45:25.982+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-05-03T07:22:07.995+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_locations permission to 509
[2025-05-03T07:22:07.996+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_locations permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_locations'
[2025-05-03T07:22:08.036+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_locations permission to 509
[2025-05-03T07:22:08.036+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_locations permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_locations'
[2025-05-03T07:22:08.080+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Airflow_ETL_DAG.insert_locations scheduled__2025-03-10T00:00:00+00:00 [queued]>
[2025-05-03T07:22:08.090+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Airflow_ETL_DAG.insert_locations scheduled__2025-03-10T00:00:00+00:00 [queued]>
[2025-05-03T07:22:08.091+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-05-03T07:22:08.103+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): insert_locations> on 2025-03-10 00:00:00+00:00
[2025-05-03T07:22:08.110+0000] {standard_task_runner.py:57} INFO - Started process 250 to run task
[2025-05-03T07:22:08.113+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'Airflow_ETL_DAG', 'insert_locations', 'scheduled__2025-03-10T00:00:00+00:00', '--job-id', '2', '--raw', '--subdir', 'DAGS_FOLDER/airflow_etl.py', '--cfg-path', '/tmp/tmpooi0r1bk']
[2025-05-03T07:22:08.116+0000] {standard_task_runner.py:85} INFO - Job 2: Subtask insert_locations
[2025-05-03T07:22:08.174+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_locations permission to 509
[2025-05-03T07:22:08.175+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_locations permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_locations'
[2025-05-03T07:22:08.176+0000] {task_command.py:416} INFO - Running <TaskInstance: Airflow_ETL_DAG.insert_locations scheduled__2025-03-10T00:00:00+00:00 [running]> on host f2b491f5d506
[2025-05-03T07:22:08.259+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='Airflow_ETL_DAG' AIRFLOW_CTX_TASK_ID='insert_locations' AIRFLOW_CTX_EXECUTION_DATE='2025-03-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-10T00:00:00+00:00'
[2025-05-03T07:22:08.948+0000] {logging_mixin.py:151} INFO - Insertion des locations terminée.
[2025-05-03T07:22:08.951+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-05-03T07:22:08.963+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=Airflow_ETL_DAG, task_id=insert_locations, execution_date=20250310T000000, start_date=20250503T072208, end_date=20250503T072208
[2025-05-03T07:22:09.009+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-05-03T07:22:09.034+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-05-05T13:47:38.242+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_locations permission to 509
[2025-05-05T13:47:38.243+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_locations permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_locations'
[2025-05-05T13:47:38.302+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_locations permission to 509
[2025-05-05T13:47:38.303+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_locations permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_locations'
[2025-05-05T13:47:38.575+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Airflow_ETL_DAG.insert_locations scheduled__2025-03-10T00:00:00+00:00 [queued]>
[2025-05-05T13:47:38.585+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Airflow_ETL_DAG.insert_locations scheduled__2025-03-10T00:00:00+00:00 [queued]>
[2025-05-05T13:47:38.586+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-05-05T13:47:38.601+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): insert_locations> on 2025-03-10 00:00:00+00:00
[2025-05-05T13:47:38.611+0000] {standard_task_runner.py:57} INFO - Started process 206 to run task
[2025-05-05T13:47:38.614+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'Airflow_ETL_DAG', 'insert_locations', 'scheduled__2025-03-10T00:00:00+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/airflow_etl.py', '--cfg-path', '/tmp/tmp43b2vw8a']
[2025-05-05T13:47:38.617+0000] {standard_task_runner.py:85} INFO - Job 3: Subtask insert_locations
[2025-05-05T13:47:38.682+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_locations permission to 509
[2025-05-05T13:47:38.683+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_locations permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_locations'
[2025-05-05T13:47:38.685+0000] {task_command.py:416} INFO - Running <TaskInstance: Airflow_ETL_DAG.insert_locations scheduled__2025-03-10T00:00:00+00:00 [running]> on host 1cb5a07b27d6
[2025-05-05T13:47:38.781+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='Airflow_ETL_DAG' AIRFLOW_CTX_TASK_ID='insert_locations' AIRFLOW_CTX_EXECUTION_DATE='2025-03-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-10T00:00:00+00:00'
[2025-05-05T13:47:39.502+0000] {logging_mixin.py:151} INFO - Insertion des locations terminée.
[2025-05-05T13:47:39.504+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-05-05T13:47:39.517+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=Airflow_ETL_DAG, task_id=insert_locations, execution_date=20250310T000000, start_date=20250505T134738, end_date=20250505T134739
[2025-05-05T13:47:39.593+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-05-05T13:47:39.623+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-05-10T17:03:02.198+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_locations permission to 509
[2025-05-10T17:03:02.199+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_locations permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_locations'
[2025-05-10T17:03:02.264+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_locations permission to 509
[2025-05-10T17:03:02.264+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_locations permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_locations'
[2025-05-10T17:03:02.387+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Airflow_ETL_DAG.insert_locations scheduled__2025-03-10T00:00:00+00:00 [queued]>
[2025-05-10T17:03:02.397+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Airflow_ETL_DAG.insert_locations scheduled__2025-03-10T00:00:00+00:00 [queued]>
[2025-05-10T17:03:02.398+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-05-10T17:03:02.453+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): insert_locations> on 2025-03-10 00:00:00+00:00
[2025-05-10T17:03:02.464+0000] {standard_task_runner.py:57} INFO - Started process 519 to run task
[2025-05-10T17:03:02.467+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'Airflow_ETL_DAG', 'insert_locations', 'scheduled__2025-03-10T00:00:00+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/airflow_etl.py', '--cfg-path', '/tmp/tmpgu33c1px']
[2025-05-10T17:03:02.471+0000] {standard_task_runner.py:85} INFO - Job 4: Subtask insert_locations
[2025-05-10T17:03:02.541+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_locations permission to 509
[2025-05-10T17:03:02.542+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_locations permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_locations'
[2025-05-10T17:03:02.544+0000] {task_command.py:416} INFO - Running <TaskInstance: Airflow_ETL_DAG.insert_locations scheduled__2025-03-10T00:00:00+00:00 [running]> on host 45059e99c3e8
[2025-05-10T17:03:02.667+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='Airflow_ETL_DAG' AIRFLOW_CTX_TASK_ID='insert_locations' AIRFLOW_CTX_EXECUTION_DATE='2025-03-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-10T00:00:00+00:00'
[2025-05-10T17:03:03.497+0000] {logging_mixin.py:151} INFO - Insertion des locations terminée.
[2025-05-10T17:03:03.499+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-05-10T17:03:03.514+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=Airflow_ETL_DAG, task_id=insert_locations, execution_date=20250310T000000, start_date=20250510T170302, end_date=20250510T170303
[2025-05-10T17:03:03.568+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-05-10T17:03:03.603+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-05-11T16:05:57.810+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_locations permission to 509
[2025-05-11T16:05:57.811+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_locations permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_locations'
[2025-05-11T16:05:57.879+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_locations permission to 509
[2025-05-11T16:05:57.879+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_locations permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_locations'
[2025-05-11T16:05:57.941+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Airflow_ETL_DAG.insert_locations scheduled__2025-03-10T00:00:00+00:00 [queued]>
[2025-05-11T16:05:57.955+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Airflow_ETL_DAG.insert_locations scheduled__2025-03-10T00:00:00+00:00 [queued]>
[2025-05-11T16:05:57.956+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-05-11T16:05:57.974+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): insert_locations> on 2025-03-10 00:00:00+00:00
[2025-05-11T16:05:57.989+0000] {standard_task_runner.py:57} INFO - Started process 261 to run task
[2025-05-11T16:05:57.993+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'Airflow_ETL_DAG', 'insert_locations', 'scheduled__2025-03-10T00:00:00+00:00', '--job-id', '2', '--raw', '--subdir', 'DAGS_FOLDER/airflow_etl.py', '--cfg-path', '/tmp/tmp7tw41qj7']
[2025-05-11T16:05:57.997+0000] {standard_task_runner.py:85} INFO - Job 2: Subtask insert_locations
[2025-05-11T16:05:58.195+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_locations permission to 509
[2025-05-11T16:05:58.196+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_locations permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_locations'
[2025-05-11T16:05:58.199+0000] {task_command.py:416} INFO - Running <TaskInstance: Airflow_ETL_DAG.insert_locations scheduled__2025-03-10T00:00:00+00:00 [running]> on host 9a7df5060863
[2025-05-11T16:05:58.400+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='Airflow_ETL_DAG' AIRFLOW_CTX_TASK_ID='insert_locations' AIRFLOW_CTX_EXECUTION_DATE='2025-03-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-10T00:00:00+00:00'
[2025-05-11T16:05:59.675+0000] {logging_mixin.py:151} INFO - Insertion des locations terminée.
[2025-05-11T16:05:59.678+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-05-11T16:05:59.696+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=Airflow_ETL_DAG, task_id=insert_locations, execution_date=20250310T000000, start_date=20250511T160557, end_date=20250511T160559
[2025-05-11T16:05:59.774+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-05-11T16:05:59.814+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-05-12T09:21:24.512+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_locations permission to 509
[2025-05-12T09:21:24.513+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_locations permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_locations'
[2025-05-12T09:21:24.580+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_locations permission to 509
[2025-05-12T09:21:24.580+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_locations permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_locations'
[2025-05-12T09:21:24.797+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Airflow_ETL_DAG.insert_locations scheduled__2025-03-10T00:00:00+00:00 [queued]>
[2025-05-12T09:21:24.804+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Airflow_ETL_DAG.insert_locations scheduled__2025-03-10T00:00:00+00:00 [queued]>
[2025-05-12T09:21:24.805+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-05-12T09:21:24.815+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): insert_locations> on 2025-03-10 00:00:00+00:00
[2025-05-12T09:21:24.821+0000] {standard_task_runner.py:57} INFO - Started process 233 to run task
[2025-05-12T09:21:24.826+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'Airflow_ETL_DAG', 'insert_locations', 'scheduled__2025-03-10T00:00:00+00:00', '--job-id', '2', '--raw', '--subdir', 'DAGS_FOLDER/airflow_etl.py', '--cfg-path', '/tmp/tmpo7x3d5oj']
[2025-05-12T09:21:24.828+0000] {standard_task_runner.py:85} INFO - Job 2: Subtask insert_locations
[2025-05-12T09:21:24.882+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_locations permission to 509
[2025-05-12T09:21:24.883+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_locations permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_locations'
[2025-05-12T09:21:24.885+0000] {task_command.py:416} INFO - Running <TaskInstance: Airflow_ETL_DAG.insert_locations scheduled__2025-03-10T00:00:00+00:00 [running]> on host 16fa90749134
[2025-05-12T09:21:24.968+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='Airflow_ETL_DAG' AIRFLOW_CTX_TASK_ID='insert_locations' AIRFLOW_CTX_EXECUTION_DATE='2025-03-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-10T00:00:00+00:00'
[2025-05-12T09:21:25.561+0000] {logging_mixin.py:151} INFO - Insertion des locations terminée.
[2025-05-12T09:21:25.564+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-05-12T09:21:25.576+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=Airflow_ETL_DAG, task_id=insert_locations, execution_date=20250310T000000, start_date=20250512T092124, end_date=20250512T092125
[2025-05-12T09:21:25.641+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-05-12T09:21:25.670+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-05-13T12:50:03.581+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_locations permission to 509
[2025-05-13T12:50:03.582+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_locations permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_locations'
[2025-05-13T12:50:03.635+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_locations permission to 509
[2025-05-13T12:50:03.635+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_locations permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_locations'
[2025-05-13T12:50:03.750+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Airflow_ETL_DAG.insert_locations scheduled__2025-03-10T00:00:00+00:00 [queued]>
[2025-05-13T12:50:03.770+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Airflow_ETL_DAG.insert_locations scheduled__2025-03-10T00:00:00+00:00 [queued]>
[2025-05-13T12:50:03.771+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-05-13T12:50:03.788+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): insert_locations> on 2025-03-10 00:00:00+00:00
[2025-05-13T12:50:03.814+0000] {standard_task_runner.py:57} INFO - Started process 649 to run task
[2025-05-13T12:50:03.827+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'Airflow_ETL_DAG', 'insert_locations', 'scheduled__2025-03-10T00:00:00+00:00', '--job-id', '2', '--raw', '--subdir', 'DAGS_FOLDER/airflow_etl.py', '--cfg-path', '/tmp/tmppkoq90ws']
[2025-05-13T12:50:03.839+0000] {standard_task_runner.py:85} INFO - Job 2: Subtask insert_locations
[2025-05-13T12:50:04.052+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_locations permission to 509
[2025-05-13T12:50:04.055+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_locations permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=Airflow_ETL_DAG/run_id=scheduled__2025-03-10T00:00:00+00:00/task_id=insert_locations'
[2025-05-13T12:50:04.059+0000] {task_command.py:416} INFO - Running <TaskInstance: Airflow_ETL_DAG.insert_locations scheduled__2025-03-10T00:00:00+00:00 [running]> on host 1d46261ea2e6
[2025-05-13T12:50:04.267+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='Airflow_ETL_DAG' AIRFLOW_CTX_TASK_ID='insert_locations' AIRFLOW_CTX_EXECUTION_DATE='2025-03-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-10T00:00:00+00:00'
[2025-05-13T12:50:05.325+0000] {logging_mixin.py:151} INFO - Insertion des locations terminée.
[2025-05-13T12:50:05.328+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-05-13T12:50:05.337+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=Airflow_ETL_DAG, task_id=insert_locations, execution_date=20250310T000000, start_date=20250513T125003, end_date=20250513T125005
[2025-05-13T12:50:05.371+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-05-13T12:50:05.395+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
